packages/web-components/src/components/link-with-icon/link-with-icon.ts

```diff
@@ -26,13 +34,19 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
  */
 @customElement(`${ddsPrefix}-link-with-icon`)
 class DDSLinkWithIcon extends StableSelectorMixin(DDSLink) {
+  /**
+   * Defines the position of the icon
+   * `left` or `right`
+   */
+  @property({ reflect: true, attribute: 'icon-placement' })
```

Comment:
For aligning dash-cased attribute name and camel-cased property name. An alternate approach is renaming `linkIconPlacement` with `iconPlacement`:

```suggestion
  @property({ reflect: true, attribute: 'link-icon-placement' })
```

Reply:
@asudoh this was addressed in my latest commit